frontend/tests/features/collection/revision.test.ts

```diff
@@ -142,26 +142,45 @@ describeIfDeployed("Collection Revision", () => {
 });
 
 async function startRevision(): Promise<string> {
-  // (thuang): NOTE: the `*` captures the "collection-row" selector
-  const COLLECTION_ROW_SELECTOR_START = `*${getTestID(
-    COLLECTION_ROW_ID
-  )} >> text="Start Revision"`;
-
   await goToPage(TEST_URL + ROUTES.MY_COLLECTIONS);
-  await page.waitForLoadState();
+  // (thuang): Wait for collections to load to prevent race condition
+  await page.waitForLoadState("networkidle");
```

Comment:
If we don't wait for network idle, Playwright will execute the rest of the code with different amounts of collections loaded on the page each time, which introduces unpredictability and flakiness